static/docs/src/openapi.yaml

```diff
@@ -30,7 +30,7 @@ parameters:
     description: |
       ID of the associated page.
 
-      **Example**: `PG_123456789012345`
+      **Example**: `PG_123456789012`
```

Comment:
Why did you shorten these?

Reply:
They are supposed to be 15 characters long, not 18.